lib/src/devices/flutterpi_ssh/device.dart

```diff
@@ -129,6 +131,7 @@ class FlutterpiSshDevice extends Device {
   final Logger logger;
   final FlutterpiCache cache;
   final MoreOperatingSystemUtils os;
+  final FlutterPiToolConfig config;
```

Comment:
See above, maybe instead introduce a `final int? rotation` member instead, just like the `explicitDevicePixelRatio` and `explicitDisplaySizeMillimeters`.